_extras/design.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+title: "Lesson Design"
+permalink: /design/
+---
+## Stage 0 - Assumptions
+
+*   Audience
+    *   Graduate students and practicing scientists in numerate disciplines from cosmology to economics
+        *   Lesson content broadly applicable across disciplines
+        *   Lesson examples from population demography ([Gapminder data][gapminder-data]) 
+    *   Must understand basic data management and manipulation 
+        *   Likely worked with spreadsheets and interactive stats tools (SAS)
+        *   Familiar with tabular data structure
+        *   Familiar with filtering and summarizing data in groups
+        *   Understands basic statistics (mean, stdev, lm) to compare groups
+    *   Should have some limited programming experience in R, including:
+        *   How to load a CSV (`read.csv()`) 
+        *   Basic `dplyr` function (`filter()`, `summarize()`, etc.)
+        *   Best preparedness benchmark from [r-novice-gapminder lesson][r-novice-gapminder]
```

Comment:
Don't understand - does this mean "we assume they've covered all the material in the r-novice-gapminder lesson"?